### PR TITLE
Fieldapriltags

### PR DESCRIPTION
**Pull request by muscatd75**

### CODE REVIEW

File: src/main/deploy/robot.xml [hunk 1]
```diff
@@ -21,27 +21,27 @@
     mountingheight="37.325"
     horizontaloffset="-7.689"
     forwardOffset="-7.24"
-    yaw="18.0"
+    yaw="17.7"
     roll="90"
     pitch="0.0"
-    targetheight="18.25"
-    targetheight2="27.25"
+    targetheight="24.125"
```
Comment (nickfrisch) [inline src/main/deploy/robot.xml:27]: Update compBot.xml targetheight and targetheight2 as well.

File: src/main/deploy/robot.xml [hunk 2]
```diff
@@ -294,20 +294,19 @@
...
   </chassis>
-
   <mechanism
     type="ARM"
     networkTable="armNT"
     controlFile="arm.xml">
     <motor
       usage="ARM"
-      canId="4"
+      canId="5"
```
Comment (nickfrisch) [inline src/main/deploy/robot.xml:307]: I don't think we should be committing the practicebot xml to main.  Switch robot.xml to be the comp bot's xml, make sure to update targetheights.

**Review by nickfrisch — CHANGES_REQUESTED** [review]
Just need to have compbot xml updated and put in robot.xml

**Comment by muscatd75** [comment]
Not all the changes can be copied to the compbot.xml

Yaw will need calibration on the compbot
Target height and targetheight2 can be copied. I can do the change tonight

**Comment by muscatd75** [comment]
Why do we not commit the practice bot xml to main?

**Comment by nickfrisch** [comment]
> Why do we not commit the practice bot xml to main?
> […](#)

Sorry I could have worded that better, robot.xml on main should contain the compbot values.

**Review by nickfrisch — APPROVED** [review]

**Pull request merged**